openwisp_controller/config/admin.py

```diff
@@ -89,9 +93,36 @@ class Meta(BaseForm.Meta):
         model = Template
 
 
+def clone_selected_templates(modeladmin, request, queryset):
+    selectable_orgs = Organization.objects if request.user.is_superuser else Organization.objects.filter(
+        pk__in=request.user.organizations_pk)
```

Comment:
`request.user.organizations_pk` is a method that already does some DB queries, so we're doing some unnecessary DB queries in this line, let's try to optimize it.

Would something like this work? Try it out please:

```python
org_count = Organization.objects.count() if request.user.is_superuser else len(request.user.organizations_pk)
```